### PR TITLE
Update README and streamline menu options for 1.5.0 release.

### PR DESCRIPTION
**Pull request by DavidSouther**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
...
+4. Select the `.vsix` file downloaded in step 1.
+
+### The Ailly extension and Bedrock
+
+When using the Bedrock engine, the Ailly extension uses your current [AWS configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
+Ailly also exposes two settings, `Ailly: AWS Profile` and `Ailly: AWS Region`, to choose which profile and (optionally) which Region to use for Bedrock requests.
+Use `aws configure` or the equivalent command to configure `~/.aws/config` and `~/.aws/credentials`.
+If Ailly shows a credentials error, check this configuration first.
+
+### Ailly Getting Starting
```
Comment (jsspric) [inline README.md:48]: "Starting" > "Started"

File: README.md [hunk 2]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
 
 Ailly's best feature is rapid prompt engineering iteration. By keeping your prompts in snippets on the file system, you can make very fine-grained changes to your prompt and immediately see the difference in the output. You can also use all of your normal source control tooling to track changes over time: both your changes, and those from the LLM.
 
+## VSCode extension
+
+The VSCode extension provides several commands.
+
+`Run` runs Ailly on a file. You can activate it with either the right-click context menu or the command palette.
```
Comment (jsspric) [inline README.md:17]: (Global) The actual commands in the context menu are "Ailly: Run," etc. so I would make sure to match the text of the commands you describe in the README to the actual text in the UI.

File: README.md [hunk 3]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
...
+
+- When you run Ailly from the context menu, it runs on the file with its parent folder as the root
```
Comment (jsspric) [inline README.md:19]: Add a period at the end of this sentence.

File: README.md [hunk 4]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
...
+- When you run Ailly from the command palette, it runs on the current active file (or does nothing if there's no active editor).
```
Comment (jsspric) [inline README.md:20]: "active editor" > "active file"

File: README.md [hunk 5]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
...
+
+`Run All` runs Ailly on a folder and all files below it in the file system.
+
+`Continue` sends the response as an `Assistant` message, to have the LLM continue generating.
+This is especially useful when a request ends before the LLM is 'finished', for instance, asking it to repeat and edit a long document.
+
+`Clean` and `Clean All` on a file or folder clean Ailly details from the file. This includes normalizing the head and removing the response contents.
```
Comment (jsspric) [inline README.md:27]: Currently, the context menu has two "Ailly: Clean" options--does this update change the second to "Ailly: Clean All"?

Comment (jsspric) [inline README.md:27]: Might want to explain what "normalizing the head" means.

File: README.md [hunk 6]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
...
+
+`Edit` runs from the command palette on the current active file (or does nothing if there's no active editor).
```
Comment (jsspric) [inline README.md:29]: Again, "active editor" > "active file" or maybe "file selected" or "file open in the editor"?

File: README.md [hunk 7]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
...
+
+When using the Bedrock engine, the Ailly extension uses your current [AWS configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
+Ailly also exposes two settings, `Ailly: AWS Profile` and `Ailly: AWS Region`, to choose which profile and (optionally) which Region to use for Bedrock requests.
+Use `aws configure` or the equivalent command to configure `~/.aws/config` and `~/.aws/credentials`.
+If Ailly shows a credentials error, check this configuration first.
+
+### Ailly Getting Starting
+
+1. Install Ailly (see above).
+2. Authenticate with Bedrock, or set the Ailly OpenAI API Key.
```
Comment (jsspric) [inline README.md:51]: "Key" > "key"

File: README.md [hunk 8]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
...
+3. Create a new folder, open it with VSCode, and create a file `10_chickens.md`.
```
Comment (jsspric) [inline README.md:52]: "create a file" > "create a file named"

File: README.md [hunk 9]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
...
+4. In the explorer view on the left, right click `10_chickens.md` and choose `Ailly: Run`.
```
Comment (jsspric) [inline README.md:53]: "right click" > "right-click"

File: README.md [hunk 10]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
...
+5. When ailly finishes, laugh at the joke in `10_chickens.md.ailly.md`!
```
Comment (jsspric) [inline README.md:54]: capitalize "Ailly"

File: README.md [hunk 11]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
...
+6. Create a file named `.aillyrc` with "You are a farmer writing jokes for your other barnyard animals."
```
Comment (jsspric) [inline README.md:55]: "with" > "that contains the following text:"

File: README.md [hunk 12]
```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
...
+7. Create a file named `20_knock_knock.md` with "Turn the chicken joke into a knock knock joke."
```
Comment (jsspric) [inline README.md:56]: "with" > "that contains the following text:"

File: README.md [hunk 13]
```diff
@@ -26,6 +73,8 @@ To get started on the command line, follow these steps:
 1. Run Ailly using NodeJS: `npx @ailly/cli 20_knock_knock.md`
    - `20_knock_knock.md.ailly.md` will have the new knock knock joke based on the chicken joke it first wrote!
```
Comment (jsspric) [inline README.md:74]: hyphenate "knock-knock"

Comment (jsspric) [reply README.md:74]: "will have" > "now contains"

**Review by jsspric — COMMENTED** [review]
Edit pass

File: README.md (hunk 5 above)
Comment (DavidSouther) [reply README.md:27]: No, the change before this did ;)

**Pull request merged**